.babelrc

```diff
@@ -1,16 +1,57 @@
 {
   "sourceMaps": true,
   "presets": [
-    ["env", {
-      "loose": true, // IE 10
-      "targets": {
-        "browsers": ["last 2 versions", "ie >= 9"]
+    "airbnb",
+    [
+      "@babel/preset-env",
+      {
+        "loose": true,
+        "targets": {
+          "browsers": [
+            "last 2 versions",
+            "ie >= 9"
+          ]
+        }
       }
-    }],
-    "stage-0"
+    ]
   ],
+  // TODO: Evaluate which Stage 0-3 proposals we're using
   "plugins": [
```

Comment:
No those are just things we need to come back to do. A better place for them may be an issue or task. The babel upgrade just split the stages into referencing their plugins directly and then I just commented them into their stages so it'd be easier to come back to

The ones in the `.eslintrc` are more helpful since they're waiting for external changes (which I may also try to contribute to)

Reply:
Gotcha, that makes sense 🌮